modules/installation-vsphere-requirements-user-provisioned.adoc

```diff
@@ -0,0 +1,258 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-requirements-user-provisioned_{context}"]
+= vCenter requirements for installations on your own infrastructure
+
+Before you install an {product-title} cluster on vCenter infrastructure that you provision, you must prepare your environment.
+
+[discrete]
+[id="installation-vsphere-requirements-user-provisioned-account_{context}"]
+== Required vCenter account privileges for installation
+
+To install an {product-title} cluster on vCenter infrastructure that you provision, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has global administrative privileges is the simplest way to access all of the necessary permissions.
+
+If you cannot use an account with global administrative privileges, you must create roles to grant the privileges necessary for {product-title} cluster installation. While most of the privileges are always required, some are required only if you plan for the installation program to provision a folder to contain the {product-title} cluster on your vCenter instance, which is the default behavior. You must create or amend vSphere roles for the specified objects to grant the required privileges.
```

Comment:
I believe you have buried the lead here. If I am understanding this correctly, the default install behavior is to provision the folder, in which case you would need global administrative privileges. My suggestion is to state that further up in the first paragraph.

Reply:
Thank you for highlighting this. This content was based on an existing module that is included in the OCP on vSphere IPI assemblies. The module includes a number of other sections that I did not carry over to this PR.

As per [this comment](https://github.com/openshift/openshift-docs/pull/39358#issuecomment-986742520), I have updated this pull request to focus specifically on adding a **Machine Management** -> **Creating machine sets** -> **Creating a machine set of vSphere** -> [**Minimum required vCenter privileges for machine set management**](https://deploy-preview-39358--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-vsphere.html#machineset-vsphere-requirements-user-provisioned-machine-sets_creating-machineset-vsphere) section, to address the requirement in [BZ 1964226](https://bugzilla.redhat.com/show_bug.cgi?id=1964226). The updated content doesn't relate to the folder creation requirement at installation time.

We can update the original module to incorporate your suggestion in a separate PR, perhaps while at the same time adding the module to the OCP on vSphere UPI modules, if applicable.